test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java

```diff
@@ -230,6 +234,57 @@ HapiApiSpec invalidTxData() {
 				).then();
 	}
 
+	// ETX-011 and ETX-030
+	HapiApiSpec originAndSenderAreEthereumSigner() {
+		return defaultHapiSpec("originAndSenderAreEthereumSigner")
+				.given(
+						newKeyNamed(SECP_256K1_SOURCE_KEY).shape(SECP_256K1_SHAPE),
+						cryptoCreate(RELAYER).balance(6 * ONE_MILLION_HBARS),
+						cryptoTransfer(tinyBarsFromAccountToAlias(GENESIS, SECP_256K1_SOURCE_KEY, ONE_HUNDRED_HBARS))
+								.via("autoAccount"),
+						getTxnRecord("autoAccount").andAllChildRecords(),
+						uploadInitCode(EMIT_SENDER_ORIGIN_CONTRACT),
+						contractCreate(EMIT_SENDER_ORIGIN_CONTRACT)
+				).when(
+						ethereumCall(EMIT_SENDER_ORIGIN_CONTRACT, "logNow")
+								.type(EthTxData.EthTransactionType.EIP1559)
+								.signingWith(SECP_256K1_SOURCE_KEY)
+								.payingWith(RELAYER)
+								.nonce(0)
+								.maxFeePerGas(50L)
+								.gasLimit(1_000_000L)
+								.via("payTxn")
+								.hasKnownStatus(ResponseCodeEnum.SUCCESS)
+				).then(
+						withOpContext((spec, opLog) -> {
```

Comment:
Rather than a hand-written opContext I think we should copy the event pattern seen here: https://github.com/hashgraph/hedera-services/blob/972c4bd6d05e93a887c1984ae346a9a87028c1c1/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/openzeppelin/ERC20ContractInteractions.java#L164

Reply:
Fixed.